plugins/registry.js

```diff
@@ -872,4 +877,19 @@ export default class PluginRegistry {
 
         return id;
     }
+
+    // INTERNAL: Subject to change without notice.
+    // Register a handler to retrieve stats that will be displayed on the system console
+    // Accepts the following:
+    // - handler - Func to be called to retrieve the stats from plugin api
+    // Returns a map with the stats.
+    registerSiteStatisticsHandler(handler) {
+        registerPluginStatsHandler(this.id, handler);
+    }
```

Comment:
This function doesn't return anything

```suggestion
    // Returns undefined.
    registerSiteStatisticsHandler(handler) {
        registerPluginStatsHandler(this.id, handler);
    }
```